src/async.tex

```diff
@@ -259,13 +259,13 @@
   &
 Shall not exit via an exception. \br\textit{Precondition:} A preceding call \tcode{x2.on_work_started()} where \tcode{x1 == x2}.  \\ \rowsep
 
-\tcode{x1.dispatch(std::move(f),a)}  &
+\tcode{x1.dispatch(\br\mbox{\ \ }std::move(f),~a)}  &
```

Comment:
This conflicts with the change in 4a0c325ab87b60710b52fbff08293330291996e9 but all that did was add the NBSP that you've also added here, so this is a better change.